charts/matrix-neoboard-widget/templates/deployment.yaml

```diff
@@ -54,7 +90,7 @@ spec:
             {{- end }}
```

Comment:
I think the all the new `env` settings are not applied to the documentation.

Reply:
i just test it again. it works